### PR TITLE
Add a sunken border to the tree window list and to the user list.

### PR DESCRIPTION
**Pull request by wodim**

### CODE REVIEW

File: src/kvirc/ui/KviTreeWindowList.cpp [hunk 1]
```diff
@@ -547,12 +557,15 @@ void KviTreeWindowListItemDelegate::paint(QPainter * p, const QStyleOptionViewIt
 			{
 				// The QWindowsXP style does not honor our colors. It uses the system ones instead.
 				// We can't accept it.
-				p->fillRect(option.rect,col);
+				QStyleOptionViewItemV4 opt4(option);
```
Comment (DarthGandalf) [inline src/kvirc/ui/KviTreeWindowList.cpp]: One local variable shadowing another local variable with the same name.
This is confusing to read.

Comment (DarthGandalf) [inline src/kvirc/ui/KviTreeWindowList.cpp]: Ah, sorry, that's in different scope

Comment (wodim) [inline src/kvirc/ui/KviTreeWindowList.cpp]: I don't even know what the `opt4` var name referred to originally but for reasons of consistency I decided to stick with it.

**Comment by un1versal** [comment]
Screenshots before and after pls?

**Comment by wodim** [comment]
Before -> look at your current KVIrc.
Now ->



The same sunken border the topic and ircview widgets have is now applied to the window list and the user list also.

**Comment by un1versal** [comment]
My current kvirc you cant tell as I dont use themes light enough  to be able to discern, and other people may not as well.

But sure +1 no brainer.

**Comment by Stanzilla** [comment]
Great change, now it finally looks the same across all panels

**Pull request merged**